src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+  display: inline-block;
+}
+
+.imgblock {
+  height: 40px;
+  width: 40px;
+}
+
+header {
+  font-family: Roboto, sans-serif;
+  height: var(--header-height);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+nav > ul > li {
+  font-style: normal;
+  align-content: center;
+  margin-left: 20px;
+}
+
+nav > ul > li:first-child {
+  margin: 0;
+}
+
+nav {
```

Comment:
The usage of tag names for styling is not recommended in CSS. It's better to style elements using `.class`. This is a code style issue. You should style elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.